src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -65,6 +65,7 @@ private sealed class State
             public readonly TaskCompletionSource ShutdownWriteCompletionSource = new TaskCompletionSource(TaskCreationOptions.RunContinuationsAsynchronously);
 
             public ShutdownState ShutdownState;
+            public int ShutdownDone;
```

Comment:
What values this can have, what do they mean and why aren't we using `ShutdownState` for that.